src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day--tue calendar--month-length--29">
+      <div class="day day--1"></div>
```

Comment:
why do we need to add modifiers to blocks? we can use nth-child

Reply:
Yeah, sure))